lib/modules/dosomething/dosomething_rogue/includes/Rogue.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use DoSomething\Gateway\Common\RestApiClient;
+use DoSomething\Gateway\ForwardsTransactionIds;
+
+class Rogue extends RestApiClient {
+  use ForwardsTransactionIds;
+
+  /**
+   * The class name of the transaction framework bridge.
+   *
+   * @var string
+   */
+  protected $transactionBridge = Phoenix\Auth\PhoenixTransactionBridge::class;
+
+  /**
+   * Create a new Rogue API client.
+   */
+  // public function __construct() {
+  //   $url = ROGUE_API_URL . '/' . ROGUE_API_VERSION . '/';
+
+  //   parent::__construct($url);
+  // }
+
+
+  /**
+   * Send a POST request of the reportback to be saved in Rogue.
+   *
+   * @param string $baseurl
+   * @param array $data
+   * @return object|false
+   */
+  public function postReportback($baseurl, $data) {
+    $response = $this->post(ROGUE_API_URL . '/' . ROGUE_API_VERSION . '/' . $baseurl, $data);
```

Comment:
@DFurnes I remember you saying that on one of Katie's old PRs but couldn't get it to work taking that out and just sending `'reportbacks'` along :-/ That is a good point though that we can just hardcode `/reportbacks` instead of sending the param here!